Model/Consumer.php

```diff
@@ -110,7 +110,11 @@ public function processMessage(MessageInterface $message)
                     return;
                 }
                 // refresh stock index to have current stock data
-                $this->indexerProcessor->reindexList([$product->getId()], true);
+                try {
+                    $this->indexerProcessor->reindexList([$product->getId()], true);
+                } catch (\Exception $exception) {
+                    // ignore elastic-suite indexer exception related to not set area code
```

Comment:
Is elastic suite indexer throwing a general Exception or some specific exception class we could catch?